README.md

```diff
@@ -67,12 +67,14 @@ Silver Sponsors are those who have pledged $2,000 to $5,000.
 
 Cats is currently available for Scala 2.10 (up to 1.2.x), 2.11, 2.12, 2.13.0, and [Scala.js](http://www.scala-js.org/).
 
-Cats relies on improved type inference via the fix for [SI-2712](https://github.com/scala/bug/issues/2712), which is not enabled by default. For **Scala 2.11.9 or later** you should add the following to your `build.sbt`:
+Cats relies on improved type inference via the fix for [SI-2712](https://github.com/scala/bug/issues/2712), which is not enabled by default. For **Scala 2.11.9 to 2.12.8** you should add the following to your `build.sbt`:
```

Comment:
Scala 2.12 might release another minor version. I'd say `For Scala 2.11.9+ or Scala 2.12`